atoMEC/models.py

```diff
@@ -175,7 +175,15 @@ def info(self):
 
     @writeoutput.timing
     def CalcEnergy(
-        self, nmax, lmax, grid_params={}, conv_params={}, scf_params={}, write_info=True
+        self,
+        nmax,
+        lmax,
+        grid_params={},
+        conv_params={},
+        scf_params={},
+        force_bound=[],
```

Comment:
Add `force_bound` to the parameters section of `CalcEnergy`'s docstring.

Reply:
See commit #84666ba